build.gradle

```diff
@@ -460,6 +461,34 @@ artifacts {
     }
 }
 
+// publishing
+publishing {
+    publications {
+        maven(MavenPublication) {
+            artifact source: jar
+            artifact source: sourcesJar, classifier: "src"
+            artifact source: devJar, classifier: "dev"
+            if (apiPackage) {
+                archives source: apiJar, classifier: "api"
```

Comment:
We have at least one case where two api artifacts are generated (second by addon.gradle). How would we integrate it into maven publish? Can we provide a hook for that or can we add a second publishing block and it will combine both?

Reply:
that's an interesting  question, not sure, let's leave it to other PR, I just wanted to do initial setup for now